package.json

```diff
@@ -31,7 +31,7 @@
     "@material/mwc-drawer": "^0.6.0",
     "@material/mwc-fab": "^0.6.0",
     "@material/mwc-formfield": "^0.6.0",
-    "@material/mwc-icon-button": "^0.6.0",
+    "@material/mwc-icon-button": "^0.11.1",
```

Comment:
btw for a future update it would be good to move to mwc-dialog etc instead of the community versions.

Reply:
Sorry? I didn't get the point, we already use mwc-dialog here.

Reply:
Not from @material !

Reply:
Do you mean use e.g. @authentic/mwc-icon-button instead in future?

Reply:
Move from authentic to material :) to the official ones

Reply:
so basically the other way around - but its a separate issue... just mentioned it as you were working on this.

Reply:
Aha, got it now. :p Will update it in future.